parm/config/config.efcs

```diff
@@ -5,19 +5,36 @@
 
 echo "BEGIN: config.efcs"
 
+# TODO: the _ENKF counterparts need to be defined in config.base
+export DO_AERO=${DO_AERO_ENKF:-"NO"}
+export DO_OCN=${DO_OCN_ENKF:-"NO"}
+export DO_ICE=${DO_ICE_ENKF:-"NO"}
+export DO_WAVE=${DO_WAVE_ENKF:-"NO"}
+
+# TODO: Possibly need OCNRES_ENKF, ICERES_ENKF, WAVRES_ENKF too
+if [[ ${DO_OCN} == "YES" ]]; then
+  case "$CASE_ENKF" in
+    "C48") export OCNRES=500;;
+    "C96") export OCNRES=100;;
```

Comment:
Is it possible that we run C96mx500?

Reply:
When we can run `C96mx500`, this block needs to be updated.  We plan to add in an option at setup time to determine the ocean resolution, rather than relying on the atmosphere.